tests/test_playbooks/repositories_role.yml

```diff
@@ -0,0 +1,35 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: fetch status
+      include_tasks: tasks/status_info.yml
+
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  roles:
+    - role: repositories
+      vars:
+        username: "{{ foreman_username }}"
+        password: "{{ foreman_password }}"
+        server_url: "{{  foreman_server_url }}"
+        validate_certs: false
+        organization: "Test Organization"
```

Comment:
this needs to be created first against `localhost`

Reply:
Done.